android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -92,6 +92,32 @@ public TwilioVoiceReactNativeModule(ReactApplicationContext reactContext) {
     MediaPlayerManager.getInstance(reactContext);
   }
 
+  /**
+   * Invoked by React Native, necessary when passing this NativeModule to the constructor of a
+   * NativeEventEmitter on the JS layer.
+   *
+   * Invoked when a listener is added to the NativeEventEmitter.
+   *
+   * @param eventName The string representation of the event.
+   */
+  @ReactMethod
+  public void addListener(String eventName) {
+    Log.d(TAG, String.format("Calling addListener: %s", eventName));
+  }
+
+  /**
+   * Invoked by React Native, necessary when passing this NativeModule to the constructor of a
+   * NativeEventEmitter on the JS layer.
+   *
+   * Invoked when listeners are removed from the NativeEventEmitter.
+   *
+   * @param count The string representation of the event.
+   */
+  @ReactMethod
+  public void removeListeners(Integer count) {
```

Comment:
Is it `removeListeners` with an `s`, or `removeListener` without an `s`? I haven't seen this method on EventEmitters https://nodejs.org/api/events.html#class-eventemitter. Please confirm that this is what you're really trying to add.

Reply:
It is indeed supposed to be plural: `removeListeners`, the docstring is wrong thanks for pointing it out haha.